application/plugins/server_alert/models/Server_alert_model.php

```diff
@@ -56,6 +56,7 @@ function add()
 			'timeout' => trim($this->input->post('timeout', TRUE)),
 			'phone_number' => trim($this->input->post('phone_number', TRUE)),
 			'respond_message' => $this->input->post('respond_message', TRUE),
+			'release_code' => '-1', // Not used for now (db requires NOT NULL)
```

Comment:
Can we use an empty string instead?

Reply:
I think we can. Shall I change to it?

Reply:
Yes please

Reply:
Works fine with ''. I pushed the change.